.gitignore

```diff
@@ -1,6 +1,5 @@
 /emacs.desktop
 *~
-*.lock
```

Comment:
I understand that this nix PR needs a lock file. Was the existing line there to prevent committing some files?

Reply:
This `*.lock` ignore was preventing creation of `flake.lock` file which happens automatically during the first execution, but maybe it is not needed anymore. Good point. Thanks.

Reply:
I wasn't thinking about removing the change, as committing changes to the flake.lock file might probably be ignored or hard to commit.
If the `*.lock` rule serves a purpose, then maybe adding a negation pattern for `flake.lock` should be used.
If the `*.lock` rules serves no purpose, then removing it makes sense.

Reply:
Updated with adding `!flake.lock` . Thanks.